.jenkins/cscs/env-gcc-cuda.sh

```diff
@@ -6,11 +6,13 @@
 
 export CRAYPE_LINK_TYPE=dynamic
 export CXX_STD="17"
+export BOOST_ROOT=/apps/dom/UES/jenkins/7.0.UP03/21.09/dom-gpu/software/Boost/1.78.0-CrayGNU-21.09
+export HWLOC_ROOT=/apps/dom/UES/jenkins/7.0.UP03/21.09/dom-gpu/software/hwloc/2.4.1/
 
 module switch PrgEnv-cray PrgEnv-gnu
-module load cudatoolkit/11.0.2_3.38-8.1__g5b73779
-module load Boost/1.75.0-CrayGNU-20.11
-module load hwloc/.2.0.3
+module switch gcc gcc/9.3.0
+module load cudatoolkit/21.5_11.3
```

Comment:
And conversely, the 11.2 module is broken with gcc. The `21.*` `cudatoolkit` modules seem to work, but have a different layout and require a newer CMake to find cusolver.